Ginger/GingerCoreNET/Application Models/Delta/APIDelta/DeltaAPIModel.cs

```diff
@@ -0,0 +1,309 @@
+﻿using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Repository;
+using GingerCore.GeneralLib;
+using GingerCoreNET.Application_Models;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GingerCoreNET.Application_Models
+{
+    public class DeltaAPIModel
+    {
+        //private bool mNewSelected = false;
+        //public bool NewSelected
+        //{
+        //    get
+        //    {
+        //        return mNewSelected;
+        //    }
+        //    set
+        //    {
+        //        mNewSelected = value;
+        //        mExistingSelected = !value;
+        //    }
+        //}
+
+        //private bool mExistingSelected = false;
+        //public bool ExistingSelected
+        //{
+        //    get
+        //    {
+        //        return mExistingSelected;
+        //    }
+        //    set
+        //    {
+        //        mExistingSelected = value;
+        //        mNewSelected = !value;
+        //    }
+        //}
+
+        private bool mIsSelected;
+        public bool IsSelected
+        {
+            get
+            {
+                return mIsSelected;
+            }
+            set
+            {
+                mIsSelected = value;
+            }
+        }
+
+        private string mName;
+        public string Name
+        {
+            get
+            {
+                return mName;
+            }
+            set
+            {
+                mName = value;
+            }
+        }
+
+        private string mDescription;
+        public string Description
+        {
+            get
+            {
+                return mDescription;
+            }
+            set
+            {
+                mDescription = value;
+            }
+        }
+
+        private string mMatchingAPIName = "";
+        public string MatchingAPIName
+        {
+            get
+            {
+                return mMatchingAPIName;
```

Comment:
why matching API model name not been pulled from actual matchingAPI object?

Reply:
Once Matching API object is set, we set it's name concatenated with it's RelativePath